tools/split_suggester.py

```diff
@@ -513,17 +609,33 @@ def userConfirms():
 
 
 # Print results
-print("Total functions: " + str(len(functions)))
-print("Total floats:    " + str(len(floatTypes)))
-print("Text Section:    0x{:X} bytes".format(textBytes))
-print("Data Section:    0x{:X} bytes".format(dataBytes))
-print("sdata2 Section:  0x{:X} bytes".format(sdataBytes))
+print("  Total functions:            " + str(len(functions)))
+print("  Total floats (in sdata2):   " + str(len(floatTypes)))
+for section, byteTotal in fileSections.items():
+    sectionName = section + " section:"
+    print("  {:28}0x{:X} bytes".format(sectionName, byteTotal))
+if uncertainBytecounts:
+    if len(uncertainBytecounts) == 1:
+        print(
+            "(The bytecount for "
+            + next(iter(uncertainBytecounts))
+            + " is uncertain, because it contains ints of unknown size.)"
+        )
+    else:
+        print(
+            "(The bytecounts for "
+            + grammarfyList(uncertainBytecounts)
+            + " are uncertain, because they contain ints of unknown size.)"
+        )
+
 if mapNames:
     print("\n\t(Function names found in the map file are shown to the right.)")
 fileCount = len(fileList)
 if debugging:
-    print("All floats data:")
+    print("\nAll floats data:")
     print(floatsData)
+    print("\nAll float types:")
+    print(floatTypes)
     print("")
```

Comment:
Please use a multiline format string literal for everything like this.

Reply:
Done.